main/Card/index.tsx

```diff
@@ -19,6 +12,11 @@ const Container = styled.View`
   shadow-offset: 0px 2px;
 `;
 
+const ImageContainer = styled.Image`
```

Comment:
i think it's better to rename it to StyledImage

Reply:
I agree. I changed its name.